src/validation.cpp

```diff
@@ -3001,6 +3003,13 @@ bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& sta
     if (block.GetBlockTime() > nAdjustedTime + 2 * 60 * 60)
         return state.Invalid(false, REJECT_INVALID, "time-too-new", "block timestamp too far in the future");
 
+    if (BIP102active(nHeight, fSegwitSeasoned)) {
+        if (block.nVersion & HARDFORK_VERSION_BIT)
+            return true;
+        return state.Invalid(false, REJECT_OBSOLETE, strprintf("bad-version(0x%08x)", block.nVersion),
```

Comment:
I copied that from [here](https://github.com/btc1/bitcoin/blob/6b0ef27b85c2093dca40a264382c336eb5727316/src/validation.cpp#L3009-L3010), guess that should be changed as well?